int32 joint_idx
float64 target
---
bool target_reached
float64 target_result
